mne/viz/_brain/colormap.py

```diff
@@ -140,7 +140,7 @@ def calculate_lut(lut_table, alpha, fmin, fmid, fmax, center=None,
         lut_table[:, -1] = np.round(lut_table[:, -1] * alpha)
 
     if divergent:
-        if fmax == fmin:
+        if np.isclose(fmax, fmin):
```

Comment:
bare `isclose` is not a great idea because it defaults to `atol=1e-08`. For something like neural currents with units of nAm (1e-9), a lot of limits will hit this. Might be better to be explict with `if np.isclose(fmax, fmin, rtol=1e-6, atol=0)`